tests/test_analysis_utils.py

```diff
@@ -47,7 +47,7 @@ def setUp(self):
             "num_cores": 1,
             "visit_in_filename": [0, 6],
             "file_format": "{0:06d}.fits",
-            "sigmaG_lims": [25, 75],
+            "sigmaG_lims": [0.0, 1.0],
```

Comment:
Shouldn't sigmaG_lims be percentiles? This would be the 0th and 1st percentile.

I think we should redesign this test to handle the percentiles. See notes below.